### PR TITLE
Fix the init method in SemanticVersion

### PR DESCRIPTION
**Pull request by Kyle-Ye**
- Fix the init method in SemanticVersion
- Update .gitignore

### CODE REVIEW

File: Sources/SymbolKit/SymbolGraph/Misc/SemanticVersion.swift [hunk 1]
```diff
@@ -42,6 +42,8 @@ extension SymbolGraph {
             self.major = major
             self.minor = minor
             self.patch = patch
+            self.prerelease = prerelease
```
Comment (Kyle-Ye) [inline Sources/SymbolKit/SymbolGraph/Misc/SemanticVersion.swift:45]: @franklinsch Is this an intended behavior? Seems like a bug for me

**Comment by Kyle-Ye** [comment]
@franklinsch Could you please help with review this PR? Thanks

File: Sources/SymbolKit/SymbolGraph/Misc/SemanticVersion.swift (hunk 1 above)
Comment (franklinsch) [reply Sources/SymbolKit/SymbolGraph/Misc/SemanticVersion.swift:45]: I agree, it seems like a bug. Is there a test we could add to exercise this code?

Comment (Kyle-Ye) [reply Sources/SymbolKit/SymbolGraph/Misc/SemanticVersion.swift:45]: I'll add a test file for it and update the PR

**Comment by Kyle-Ye** [comment]
Added the test file and a simple test function to exercise the fixed-code. @franklinsch

**Comment by franklinsch** [comment]
@swift-ci please test

**Comment by franklinsch** [comment]
@swift-ci please test

**Review by franklinsch — APPROVED** [review]
Thanks @Kyle-Ye! 🙌

**Pull request merged**